gamemodes/terrortown/gamemode/shared/sh_door.lua

```diff
@@ -113,7 +125,9 @@ local entmeta = assert(FindMetaTable("Entity"), "FAILED TO FIND ENTITY TABLE")
 -- @return boolean Returns true if it is a valid door
 -- @realm shared
 function entmeta:IsDoor()
-	if IsValid(self) and door.__valid_doors[self:GetClass()] then
+	local valid_door_cls = door.GetValid()
```

Comment:
why did you localized it?

Reply:
to access it with `[self:GetClass()]`

Reply:
`door.GetValid()[self:GetClass()]` wouldn't work? ^^

Reply:
I haven't tested it. I thought it wouldn't^^ I can test it